src/main/java/com/example/eventorganizer/event/EventController.java

```diff
@@ -39,15 +38,18 @@ public ModelAndView getAll(){
         mav.addObject("events",eventRepository.findAll());
         return mav;
     }
-
+    @GetMapping(value="/home/inviteUser")
```

Comment:
If you use multiple words,. it should be `invite-user`. Please take a look at url naming conventions and update your endpoints.
https://restfulapi.net/resource-naming/